### PR TITLE
Fix: Fixes before deployment

### PR DESCRIPTION
**Pull request by cp-priyanka-g**
## Review checklist

**Note:** Make sure all points should be checked before submiting PR for review, otherwise PR will be cosidered invalid.

- [x] Follow guidelines for creating branch name, adding commit messages or PR title
- [ ] Add MR description if applicable
- [x] Follow guidelines for naming conventions everywhere (i.e files/folders, data structures)
- [ ] Reuse code (if the same code is written twice, make it common and reuse it at both places)
- [ ] Remove unused or commented code if not required
- [x] Make MR self-approved - review MR as a reviewer and give it self-approval if everything is ok, if not make the required changes
- [ ] In progress MR should be marked as draft
- [x] Make MR mark as ready before submitting it for review
- [ ] Add images, videos, or gifs of UI 
- [ ] Add page speed score screenshot of UI (It should be > 85 for mobile and > 90 for desktops)

**Design should be tested on:**
- [x] Inspector (all iPhones and android devices like samsung, motorola)
- [x] Desktop (chrome, firefox, safari)
- [ ] Laptop / Macbook  (chrome, firefox, safari)
- [ ] iPhone(safari)
- [ ] Available android devices  (chrome, firefox)
- [ ] iPad(safari)
- [ ] Tablet (chrome, firefox)

**Description:**
- [x] Update all the stars, claps with real-time values on the whole page
            1. Trending section: fix the hover effect
            2. Only the hovered card's row should be paused
- [x] fix padding to solve line issue in 560 widths.
- [x] Github section: Reduce space between tabs and images
- [x] Reduce the width of weekly tabs in lg devices - it shouldn't exceed the card width for all devices (Attached screenshot)
- [x] Weekly section: remove the title bar from the mobile view.
- [x]  Unit test section :Increase Div size in 4K devices.
- [x] Favorite section: Title should be in one line after sm device.
- [x] Fix the career page description & remove horizontal padding.
- [x] The hover effect is not working properly in The Favourites You Can't Miss Section.
- [x] The weekly tech section position should be fixed when scrolling up or down
- [x] Fix animation of the design section:
            1.Design section: When clicking on animation, the animation will be paused. We need to refresh the page to again start the animation.
           2. Cards should continue to be resized when changing the position instead of resize after changing the position.

### CODE REVIEW

File: vue-frontend/src/components/contributions/Favourite.vue [hunk 1]
```diff
@@ -1,11 +1,10 @@
 <template>
-  <section class="tw-my-20 md:tw-mt-16">
+  <section class="tw-my-20 md:tw-mt-16 tw-overflow-hidden">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/Favourite.vue:2]: The effect is still working on hovering outside of the image

File: vue-frontend/src/components/contributions/DevOps.vue [hunk 2]
```diff
@@ -1,6 +1,6 @@
 <template>
   <section
-    class="tw-mb-[100px] tw-mt-[12%] lg:tw-mt-[7%] 2xll:tw-mt-[4%] 3xl:tw-mt-0"
+    class="tw-mb-[100px] tw-mt-[12%] lg:tw-mt-[7%] 2xll:tw-mt-[4%] 3xl:tw-mt-0 tw-overflow-hidden"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DevOps.vue:3]: Not working

File: vue-frontend/src/components/contributions/DesignExplore.vue [hunk 3]
```diff
@@ -54,16 +54,14 @@
         <swiper
           :slidesPerView="3"
           :centeredSlides="true"
+          :loop="true"
+          :loopedSlides="50"
           :autoplay="{
-            delay: 2000,
+            delay: 3000,
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DesignExplore.vue:60]: I think its too slow

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/tailwind.config.cjs [hunk 4]
```diff
@@ -153,8 +153,8 @@ module.exports = {
         gridAnimationReverse: "scroll-reverse 20s linear infinite",
         gridAnimationReverse2: "scroll-reverse 40s linear infinite",
         fadeIn: "fadeIn 0.6s ease-in",
-        verticalZoom: "verticalZoom 4s  ease-in-out forwards",
-        verticalZoomOut: "verticalZoomOut 4s  ease-in-out forwards",
+        verticalZoom: "verticalZoom 2s  ease-in-out forwards",
```
Comment (cp-sumi-k) [inline vue-frontend/tailwind.config.cjs:156]: Any use of it?

File: vue-frontend/src/components/contributions/DesignExplore.vue [hunk 5]
```diff
@@ -320,9 +319,10 @@ export default {
 @import "swiper/css";
 @import "swiper/css/pagination";
 @import "swiper/css/free-mode";
+@import "swiper/css/zoom";
 
 .swiper-slide-active .image-content {
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DesignExplore.vue:324]: Why need to manage bottom margin from here?

File: vue-frontend/src/components/contributions/AnimatedCreation.vue [hunk 6]
```diff
@@ -43,7 +43,7 @@
       <div class="tw-flex tw-flex-col tw-gap-8 tw-w-[50%]">
         <div v-for="animation in animations.slice(0, 3)" :key="animation.id">
           <div
-            class="tw-relative tw-group tw-cursor-pointer"
+            class="tw-relative tw-group tw-cursor-pointer tw-overflow-hidden"
             @click="openBlog(animation)"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/AnimatedCreation.vue:47]: I think this section not contaning any blog

File: vue-frontend/src/components/contributions/DesignExplore.vue [hunk 7]
```diff
@@ -85,17 +85,16 @@
...
             disableOnInteraction: false,
           }"
+          :speed="3000"
           :spaceBetween="20"
           :navigation="true"
-          :loop="true"
-          :loopedSlides="50"
           class="swiper-container"
         >
           <swiper-slide v-for="(post, index) in data.slice(5, 10)" :key="index">
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/contributions/DesignExplore.vue:100]: post is not relative name for this section

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: vue-frontend/src/components/contributions/AnimatedCreation.vue (hunk 6 above)
Comment (cp-priyanka-g) [reply vue-frontend/src/components/contributions/AnimatedCreation.vue:47]: yes, its blog only.

File: vue-frontend/src/components/contributions/WeeklyUpdate.vue [hunk 8]
```diff
@@ -86,9 +66,9 @@
     <!-- Mobile UI end -->
     <!-- Desktop UI start -->
     <div
-      class="tw-hidden md:tw-flex tw-justify-around tw-m-auto tw-mb-[7rem] tw-mt-8 md:tw-my-[3.75rem] tw-w-[100%] md:tw-w-[95%] xl:tw-w-[65%] 2xl:tw-w-[55%] xll:tw-w-[30%] tw-rounded-[30px] tw-py-[5px]"
+      class="tw-hidden md:tw-flex tw-justify-around tw-m-auto tw-mb-[7rem] tw-mt-8 md:tw-my-[3.75rem] tw-w-[100%] md:tw-w-[95%] lg:tw-w-[77%] xl:tw-w-[65%] 2xl:tw-w-[55%] xll:tw-w-[30%] tw-rounded-[30px] tw-py-[5px]"
     >
-      <a
+      <div
```
[the remark this answers is not in the document]
Comment (cp-priyanka-g) [reply vue-frontend/src/components/contributions/WeeklyUpdate.vue:71]: As there was no, href in <a> tag, so in lighthouse SEO score was down "uncrawalable links". so i change it to div. But i will revert the change add the href to this links.

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**